src/devices/sara_u280.c

```diff
@@ -40,10 +40,12 @@
 #define GPRS_ATTACH_ATTEMPTS	5
 #define GPRS_ATTACH_BACKOFF_MS	1000
 #define NET_REG_ATTEMPTS	50
+#define SARA_U280_ERROR_THRESHOLD 6
 #define NET_REG_BACKOFF_MS	1000
 #define STOP_DM_RX_EVENTS	10
 #define STOP_DM_RX_TIMEOUT_MS	1000
 #define SARA_U280_SUBSCRIBER_NUMBER_RETRIES 3
+#define FAILED_READ_BACKOFF_MS 100
```

Comment:
That is quite some time.  Why this number?

Reply:
Eh, you're right - it seems arbitrary. There is no evidence we even need this delay so I will remove it.

Reply:
Ok, removed delay.